### PR TITLE
[IMP] developer: external api add admonition block

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task: https://www.odoo.com/web#id=3703727&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

File: content/developer/reference/external_api.rst [hunk 1]
```diff
@@ -12,6 +12,9 @@ easily available over XML-RPC_ and accessible from a variety of languages.
    Check out the `manual <https://www.php.net/manual/en/xmlrpc.installation.php>`_
    for the installation steps.
 
+.. note::
+   Access to data via the external API is only available on odoo.SH and on-premise databases.
```
Comment (xpl-odoo) [inline content/developer/reference/external_api.rst:16]:
```suggestion
   Access to data via the external API is only available on Odoo.sh and on-premise databases.
```
While no Odoo style guide, Odoo.sh is stylized this way usually.

**Review by AntoineVDV — APPROVED** [review]

**Comment by jam-odoo** [comment]
Hello @tiku-odoo ,
As far as I know, the API is available on all hosting options (SaaS/Sh/Self) with Custom Plan. Standrd Plan does not allow access to API on Odoo Online, our pricing same states same https://www.odoo.com/pricing-plan

Here are some [code heuristics added by the internal](https://github.com/odoo/internal/commit/52f6eae94cc35e291f17e179d0d853f551d30c78) team to Odoo SaaS Platform for tapping the API calls

**Comment by tiku-odoo** [comment]
@jam-odoo 
Thanks for your expertise. I've drafted changes to reflect the custom plan that has access to the external API. 

@xpl-odoo 
This is ready for your review when you have a moment.

Thanks, 
Tim

 CC: @StraubCreative

**Review by xpl-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**